libraries/c_sdk/aws/shadow/src/aws_shadow.c

```diff
@@ -80,33 +80,33 @@ typedef struct ShadowClient
 /*-----------------------------------------------------------*/
 
 /**
- * @brief Converts a Shadow v4 return code to a Shadow v1 return code.
+ * @brief Converts a Shadow v2 return code to a Shadow v1 return code.
  *
- * @param[in] xShadowError The Shadow v4 return code.
+ * @param[in] xShadowError The Shadow v2 return code.
  *
  * @return An equivalent Shadow v1 return code.
  */
 static ShadowReturnCode_t prvConvertReturnCode( AwsIotShadowError_t xShadowError );
 
 /**
- * @brief Wrapper for Shadow v4 delta callbacks. Invokes the Shadow v1 callback.
+ * @brief Wrapper for Shadow v2 delta callbacks. Invokes the Shadow v1 callback.
  *
  * @param[in] pvArgument The associated #ShadowClient_t.
- * @param[in] pxUpdatedDocument Shadow v4 delta document info.
+ * @param[in] pxUpdatedDocument Shadow v2 delta document info.
  */
 static void prvDeltaCallbackWrapper( void * pvArgument,
                                      AwsIotShadowCallbackParam_t * const pxDeltaDocument );
 
 /**
- * @brief Wrapper for Shadow v4 updated callbacks. Invokes the Shadow v1 callback.
+ * @brief Wrapper for Shadow v2 updated callbacks. Invokes the Shadow v1 callback.
  *
  * @param[in] pvArgument The associated #ShadowClient_t.
- * @param[in] pxUpdatedDocument Shadow v4 updated document info.
+ * @param[in] pxUpdatedDocument Shadow v2 updated document info.
  */
 static void prvUpdatedCallbackWrapper( void * pvArgument,
                                        AwsIotShadowCallbackParam_t * const pxUpdatedDocument );
 
-/* Retrieves the MQTT v4 connection from the MQTT v1 connection handle. */
+/* Retrieves the MQTT v2 connection from the MQTT v1 connection handle. */
 extern IotMqttConnection_t MQTT_AGENT_Getv4Connection( MQTTAgentHandle_t xMQTTHandle );
```

Comment:
Note: I didn't change the function name here

Reply:
This can be changed as it's a function internal to the conversion layer. (The only reason the function isn't static is because it is used in both the MQTT and Shadow conversion layers.)